src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -653,6 +691,68 @@ public function resendEmailAction(int $orderId, int $orderStatusId, int $orderHi
         ]);
     }
 
+    /**
+     * @param int $orderId
+     * @param int $orderDetailId
+     *
+     * @return JsonResponse
+     */
+    public function deleteProductAction(int $orderId, int $orderDetailId): JsonResponse
+    {
+        try {
+            $this->getCommandBus()->handle(
+                new DeleteProductFromOrderCommand($orderId, $orderDetailId)
+            );
+
+            return $this->json(null);
```

Comment:
If you plan on returning an empty response better use a 204 response (no content but valid response)

Reply:
:ballot_box_with_check: